Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/ConsoleLoggerWriter.cs

```diff
@@ -217,31 +230,35 @@ internal void FormattedWriteLine(string message)
 
             internal void FormattedWriteLine(string level, string message)
             {
-                var displayLevel = level;
-                if (Enum.TryParse<LogLevel>(level, true, out var levelEnum))
+                lock(LockObject)
                 {
-                    if (levelEnum < _minmumLogLevel)
-                        return;
+                    var displayLevel = level;
+                    if (Enum.TryParse<LogLevel>(level, true, out var levelEnum))
+                    {
+                        if (levelEnum < _minmumLogLevel)
+                            return;
 
-                    displayLevel = ConvertLogLevelToLabel(levelEnum);
-                }
+                        displayLevel = ConvertLogLevelToLabel(levelEnum);
+                    }
 
-                if (_logFormatType == LogFormatType.Unformatted)
-                {
-                    _innerWriter.WriteLine(message);
-                }
-                else
-                {
-                    string line;
-                    if (!string.IsNullOrEmpty(displayLevel))
+                    if (_logFormatType == LogFormatType.Unformatted)
                     {
-                        line = $"{DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.fffZ")}\t{CurrentAwsRequestId}\t{displayLevel}\t{message ?? string.Empty}";
+                        _innerWriter.WriteLine(message);
                     }
                     else
                     {
-                        line = $"{DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.fffZ")}\t{CurrentAwsRequestId}\t{message ?? string.Empty}";
+                        string line;
```

Comment:
This is mostly whitespace changes due to the new indent of the lock. If you click hide whitespace changes you can see there is very little change.